pkg/chain/celo/tbtc.go

```diff
@@ -452,7 +452,24 @@ func (ta *tbtcApplication) FundingInfo(
 		return nil, err
 	}
 
-	transactionHash, outputIndex := chain.ParseUtxoOutpoint(fundingInfo.UtxoOutpoint)
+	logger.Debugf(
+		"deposit [%s] funding info: %+v",
+		depositAddress,
+		fundingInfo,
+	)
+
+	if fundingInfo.FundedAt == nil || fundingInfo.FundedAt.Cmp(big.NewInt(0)) <= 0 {
```

Comment:
when would `FundedAt` be negative?

Reply:
> when would FundedAt be <= 0?

We assume a deposit is funded when `FundedAt > 0`. Here we need to check the opposite, so I used `<= 0` just in case we get a negative value from the chain.

> edit: also, does fundingInfo.FundedAt <= big.NewInt(0) work here?

Nope, we need to use `Cmp` function to compare `big.Int`s.